underfs/hdfs/pom.xml

```diff
@@ -149,6 +165,7 @@
         </plugins>
       </build>
     </profile>
+
```

Comment:
NIT: not necessary?

Reply:
i added the empty line because we use an empty line to separate the profiles from each other. This makes profiles stand out more.